_events/2022/2022-08-dei-speaker-series.md

```diff
@@ -0,0 +1,73 @@
+community at large. Our community pledges to continually educate ourselves
+through research on best practices focused on diversity, equity, and inclusion.
+US-RSE is committed to providing an inclusive environment with equitable
+treatment for all and to promoting and encouraging diversity throughout the RSE
+community. We encourage everyone to suggest speakers and topics and are open to
+different formats and timelines of the events. Feedback is much appreciated.
+
+
+## Event
+
+The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is proud to
+welcome Vandana Singh to present her talk "Diversity in Open Source Software."
+This event will take place
+**Wednesday, Aug 31st at 1 PM PT / 2 PM MT / 3 PM CT / 4 PM ET**
+
+[**Register on Zoom**](https://emory.zoom.us/meeting/register/tJEsceiqrTwuEt3imLDh_7DNURubgHPKC4Tk)
+
+### Abstract
+
+Dr. Vandana Singh’s research investigates the experiences of marginalized
+groups within technical fields like software engineering and information
+technology. In this discussion, Dr. Singh will start by discussing the current
+status of Diversity in Open Source Software (OSS), especially with respect to
+women of OSS and their experiences of being in OSS communities. Dr. Singh will
+share some results from her research on this topic and will discuss recent
+events pertinent to Diversity in OSS. Then, we will look at some current
+initiatives that address this challenge and together identify what more can be
+done. What are the glaring gaps, what should the organizations be doing to
+support women in OSS? We will conclude with setting intentions for a stronger
+path forward for empowering women and women leaders in OSS with concrete
+recommendations for communities and guidelines for women to be successful in
+OSS.
+
+### Biography
+
+Dr. Vandana Singh is a Professor at the School of Information Sciences, and
+Director of Diversity, Equity, and Inclusion at the College of Communication
+and Information at the University of Tennessee- Knoxville. Her research focuses
+on the use of technology in various settings and includes Gender and
+Information Technology; Open Source Software; Diversity, Equity, and Inclusion;
+Online Learning; Rural Libraries and Technology; STEM education. Her research
+is funded by federal agencies such as National Science Foundation (NSF), IMLS,
+and the United States Geological Society (USGS). Her research is published in
+top information science journals, national and international conferences, and
+books. She teaches Information Technology courses, including Web Design,
+Usability Testing, IT Foundations, and DEI for Technology/Information
+Professionals. Currently, Singh serves as an Executive Committee Member for
+Association for Computing Machinery-Women and the programming chair for the
+National Center for Women & Information Technology Academic Alliance. To know
+more about Dr. Singh, please visit her website at
+
```

Comment:
this blank line should be removed to make the formatting better

Reply:
@danielskatz Sure thing. I'll fix that now. Other than this, do you think it is good to merge?

Reply:
yes